### ISSUE TITLE
Sundry editions improvements

### ISSUE THREAD

**Issue by holtchesley** [harvard-lil/h2o#1375]
Admin page for editing + adding casebooks, changing name

**Comment by holtchesley** [comment]
Easily change name of group after creation.

**Comment by cath9** [comment]
Consider removing all but most recent version from search results.

**Comment by cath9** [comment]
"Delete Group" could lead to the question of whether that would delete the contents of the group as well (ie the books). Maybe reword to "Undo Grouping"?

**Comment by cath9** [comment]
@lizadaly benefits to the grouping option are largely to help clean up an author's dashboard and to clean up search. I think its a needed feature that is currently underused. We could improve the UI in ways suggested above, and also consider doing more to prompt people to arrange their books into series. ie if somebody clones their own book, there could be a prompt that asks them if they want to group those books into a series, or something like that.

**Comment by lizadaly** [comment]
I've split these out into individual tickets with the `editions` tag: https://github.com/harvard-lil/h2o/labels/editions
